tests/test_astrometryModel.py

```diff
@@ -37,10 +38,13 @@
 import lsst.afw.image.utils
 import lsst.daf.persistence
 import lsst.geom
+import lsst.log
+
 import lsst.jointcal
+from lsst.jointcal.jointcal import make_schema_table, get_sourceTable_visit_columns, \
+    extract_detector_catalog_from_visit_catalog
```

Comment:
I thought there was some preference for using from blah import (blah, blah, blah) when they went over a line but having read a bunch of things I can't find it. I'm adding this comment in case it triggers either of us to remember where I got this instruction in the past.

Reply:
I don't see it mentioned in the dev guide anywhere either, but that's reasonable.